src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -50,3 +49,10 @@ void uct_rdmacm_cm_ep_error_cb(uct_rdmacm_cm_ep_t *cep,
 
 const char* uct_rdmacm_cm_ep_str(uct_rdmacm_cm_ep_t *cep, char *str,
                                  size_t max_len);
+
+void uct_rdmacm_cm_ep_client_connect_cb(uct_rdmacm_cm_ep_t *cep,
```

Comment:
@evgeny-leksikov these functions were removed in another PR. did you add them intentionally or merging mistake?

Reply:
Added intentionally to avoid code duplication with flags setting, now these functions invoke common code added by you in that PR.

Reply:
but if this PR is merged then these rdmacm functions will be added to the repo and they shouldn't be there anymore - they were removed.

Reply:
What do you mean? I added them intentionally with new implementation.

Reply:
you mean with the addition of the UCT_RDMACM_CM_EP_CONNECTED flag setting?

Reply:
yes